### PR TITLE
Paw paw

### PR DESCRIPTION
**Pull request by ayan-IN**
**Changes / Implementations :**
1. Hero Section added - contains basic navigation part and the hero context.
2. Secondary Section - contains additional information about the breeds.
3. Call to action section - contains information and button to sign up
4. Footer Section - contains copyright and year information

Demo Link : [https://ayan-in.github.io/paw-paw/](https://ayan-in.github.io/paw-paw/)

_N.B : Flex property is used in each and every section in some way to get the required output._

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+.external-links > a {
+  color: #e5e7eb;
+  text-decoration: none;
+  color: white;
+  font-size: 20px;
+}
+
+/* Header Section */
+
```
Comment (sankar-jtc) [inline styles.css:49]: Remove this line.
Try to follow the same practices for indentation.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+  background-color: #3882f6;
+  color: white;
+  width: 130px;
+  font-size: 18px;
+  padding: 10px;
+  border-radius: 10px;
+}
+
+/* Secondary Header Section */
+
```
Comment (sankar-jtc) [inline styles.css:92]: Remove this line.
Try to follow the same practices for indentation.

**Review by sankar-jtc — APPROVED** [review]

**Comment by sankar-jtc** [comment]
Please address the comment and merge.

**Comment by ayan-IN** [comment]
Addressed issue resolved. Merging.

**Pull request merged**